public/sample.csv

```diff
@@ -1,4 +1,5 @@
-token_address,receiver,amount
-0x6810e776880c02933d47db1b9fc05908e5386b96,0x1000000000000000000000000000000000000000,0.0001
-0x6b175474e89094c44da98b954eedeac495271d0f,0x2000000000000000000000000000000000000000,0.0001
-,0x3000000000000000000000000000000000000000,0.0001
\ No newline at end of file
+token_type,token_address,receiver,value,id
+erc20,0x6810e776880c02933d47db1b9fc05908e5386b96,0x1000000000000000000000000000000000000000,0.0001
+erc20,0x6b175474e89094c44da98b954eedeac495271d0f,0x2000000000000000000000000000000000000000,0.0001
+native,,0x3000000000000000000000000000000000000000,0.0001
+erc721,0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85,0x4000000000000000000000000000000000000000,,42
```

Comment:
Does this particular NFT exist on mainnet? Just want to keep the sample file something that could actually work there.

Reply:
I think this is the ENS ERC721 contract.